pkg/observer/observer_stats.go

```diff
@@ -90,4 +90,16 @@ func (k *Observer) startUpdateMapMetrics() {
 			}
 		}
 	}()
+
+	// Map loaded sensor map metrics
+	go func() {
+		mapmetrics.UpdateSensorMapsLoaded()
+		ticker := time.NewTicker(1 * time.Minute)
+		for {
+			select {
```

Comment:
nit: any reason why we cannot add a single `UpdateSensorMapsLoaded` here?

Reply:
@kkourt we settled on this approach when I was discussing it with @jrfastab, I think mostly because the other way might be error-prone in unexpected ways. The point of this metric is to catch weirdness with sensor loading/unloading and making it dependent on the sensor loading/unloading logic might hide some things. Does that make sense?

Reply:
About calling it twice, I just wanted to get metrics right away before the initial minute is up.

Reply:
I think we can have both, no?

```
ticker := time.NewTicker(1 * time.Minute)
for {
    // do thing
    select {
       // wait
    }
}
```

Reply:
Ah good point!

Reply:
> The point of this metric is to catch weirdness with sensor loading/unloading and making it dependent on the sensor loading/unloading logic might hide some things. Does that make sense?

(sorry, missed this in my first reply)

It makes sense, yes.

On my side, I prefer the approach where we hook into the functions we use. Some arguments:
 * Checking every 1' might tell you when something is wrong, but it's also imprecise because there is a time window between when things went wrong, and when we see it. The time window is small, so maybe this does not matter much.
 * If there are ways were things can be loaded via paths we do not understand, then maybe a refactoring make sense to ensure that we can track the ways sensors are loaded.

That being said, I do not object to this PR as it is right now. I was just trying to ensure that the alternative was considered. If it was, then LGTM.

NB: I'm seeing a coupe of new patches, but I haven't checked those.   Maybe they aim to combine the two approaches? Not sure :)